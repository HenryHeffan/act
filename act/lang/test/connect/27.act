bool! x;
bool? y;

x = y;
