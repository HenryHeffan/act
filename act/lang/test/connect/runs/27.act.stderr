In expanding ::<Global>
Connecting `x' and `y' failed.
	Type 1: bool! [dir=!]
	Type 2: bool? [dir=?]
FATAL: Illegal combination of types
